mscore/menus.cpp

```diff
@@ -1503,51 +1504,51 @@ PalettePanel* MuseScore::newTempoPalettePanel(bool defaultPalettePanel)
+            TempoPattern("Adagio",           "Adagio",            71.0/60.0, false, true, false, false, false),
+            TempoPattern("Andante",          "Andante",           92.0/60.0, false, true, false, false, false),
+            TempoPattern("Andantino",        "Andantino",         94.0/60.0, false, true, false, false, true),
+            TempoPattern("Moderato",         "Moderato",         114.0/60.0, false, true, false, false, false),
+            TempoPattern("Allegretto",       "Allegretto",       116.0/60.0, false, true, false, false, false),
+            TempoPattern("Allegro moderato", "Allegro moderato", 118.0/60.0, false, true, false, false, true),
+            TempoPattern("Allegro",          "Allegro",          144.0/60.0, false, true, false, false, false),
+            TempoPattern("Vivace",           "Vivace",           172.0/60.0, false, true, false, false, false),
+            TempoPattern("Presto",           "Presto",           187.0/60.0, false, true, false, false, false),
+            TempoPattern("Prestissimo",      "Prestissimo",      200.0/60.0, false, true, false, false, true),
+
+            TempoPattern("<sym>metNoteQuarterUp</sym> = <sym>metNoteQuarterUp</sym><sym>space</sym><sym>metAugmentationDot</sym>", "Quarter note = dotted quarter note metric modulation", 3.0/2.0, true, false, true, false, false),
+            TempoPattern("<sym>metNoteQuarterUp</sym><sym>space</sym><sym>metAugmentationDot</sym> = <sym>metNoteQuarterUp</sym>", "Dotted quarter note = quarter note metric modulation", 2.0/3.0, true, false, true, false, false),
+            TempoPattern("<sym>metNote8thUp</sym><sym>space</sym><sym>metAugmentationDot</sym> = <sym>metNoteQuarterUp</sym>",     "Dotted eighth note = quarter note metric modulation",  2.0/3.0, true, false, true, false, false),
+            TempoPattern("<sym>metNoteHalfUp</sym> = <sym>metNoteQuarterUp</sym>",    "Half note = quarter note metric modulation",    1.0/2.0, true, false, true, false, false),
+            TempoPattern("<sym>metNoteQuarterUp</sym> = <sym>metNoteHalfUp</sym>",    "Quarter note = half note metric modulation",    2.0/1.0, true, false, true, false, false),
+            TempoPattern("<sym>metNote8thUp</sym> = <sym>metNote8thUp</sym>",         "Eighth note = eighth note metric modulation",   1.0/1.0, true, false, true, false, false),
+            TempoPattern("<sym>metNoteQuarterUp</sym> = <sym>metNoteQuarterUp</sym>", "Quarter note = quarter note metric modulation", 1.0/1.0, true, false, true, false, false),
             };
       for (TempoPattern tp : tps) {
             TempoText* tt = new TempoText(gscore);
             tt->setFollowText(tp.followText);
             tt->setXmlText(tp.pattern);
             if (tp.relative) {
                   tt->setRelative(tp.f);
-                  sp->append(tt, QT_TRANSLATE_NOOP("Palette", "Metric modulation"), QString(), 1.5);
+                  sp->append(tt, QT_TRANSLATE_NOOP("Palette", tp.name), QString(), 1.5);
```

Comment:
I don't think this and the next 2 work. Don't think those need to get translated either. "Lento" is "Lento in any language as is "<sym>metNoteHalfUp</sym> = <sym>metNoteQuarterUp</sym>" or "<sym>metNoteHalfUp</sym> = 80"

Reply:
But if needed, the QT_TRANSLATE_NOOP needs to be in the array, not here

Reply:
All right, I added `QT_TRANSLATE_NOOP` in the array, and changed this one to `qApp->translate()`. I left the Italian names as they are, didn't add the translation function to them.

Also I just read the QT translation documentations, and now I understood what these functions actually do. I should have probably read them earlier, lol.